README.md

```diff
@@ -91,6 +91,12 @@ options:
                         Init image
   --mask_image MASK_IMAGE, -m MASK_IMAGE
                         Mask image
+  --upscale_engine UPSCALE_ENGINE, -U UPSCALE_ENGINE
```

Comment:
(non blocking feature suggestion for future PR): i feel like default engines should be configurable as environment variables. thoughts?

Reply:
Agreed this would be a nice option. Submitted a feature request https://github.com/Stability-AI/stability-sdk/issues/238